cranelift/fuzzgen/src/lib.rs

```diff
@@ -40,17 +40,34 @@ target x86_64
 
         writeln!(f, "{}", self.func)?;
 
+        writeln!(f, "; Note: the results in the below test cases are simply a placeholder and probably will be wrong\n")?;
+
         for input in self.inputs.iter() {
+            // TODO: We don't know the expected outputs, maybe we can run the interpreter
+            // here to figure them out? Should work, however we need to be careful to catch
+            // panics in case its the interpreter that is failing.
+            // For now create a placeholder output consisting of the zero value for the type
+            let placeholder_output = input
```

Comment:
I'm confused. Shouldn't this look at the function's signature to get the return type, rather than looking at the test inputs?

Other than that, this seems great. I like the way you're constructing the placeholders.